tests/destinations/test_custom_destination.py

```diff
@@ -437,6 +451,54 @@ def my_gcp_sink(
     )
 
 
+def test_destination_with_spec() -> None:
```

Comment:
lmk if this is enough wrt to testing the custom base

Reply:
improved a little bit. spec is not a custom base. it is THE spec that will be used